_episodes/#01-basics.md#

```diff
@@ -0,0 +1,57 @@
+---
+title: Basics of Testing
+teaching: 5
+exercises: 0
+questions:
+- "Why would you not want to test?"
+objectives:
+- "Understand the place of testing in a scientific workflow."
+- "Understand that testing has many forms."
+keypoints:
+- "Tests compare that the result observed from running code is the same as what was expected ahead of time."
```

Comment:
Rephrase for clarity "Tests check whether the observed result, from running the code, is what was expected ahead of time."